InactiveIssue/Get_Inactive_Issue.py

```diff
@@ -0,0 +1,64 @@
+import time
+import json
+
+from github import Github
+from dateutil.parser import parse
+from datetime import date
+from django.http import HttpResponse
+from gci.config import get_api_key
+
+
+def run(issue1):
+    issues_number_list = []
+    for j in issue1:
+        issue_no = j.number
+        events = j.get_events()
+        myevent = []
+        data = []
+        for i in events:
+            myevent.append(str(i.event))
+        for i in events:
+            if i.commit_id is not None:
+                data.append(str(i.created_at))
+        for i, myevents in reversed(list(enumerate(myevent))):
+            if myevents == 'unassigned':
+                break
+            elif myevents == 'assigned':
+                a = events[i].created_at
+                c = (date.fromtimestamp(time.time()) - a.date()).days
+                if c >= 60:  # for checking assigned duration
+
+                    mydata = list(reversed(data))
+                    if len(mydata) != 0:
+                        commit1 = parse(mydata[0])
+                        calculated_days = (date.fromtimestamp(
+                            time.time()) - commit1.date()).days
+                        if calculated_days >= 60:
+                            # for checking last commit update
+                            issues_number_list.append(issue_no)
+                    else:
+                        issues_number_list.append(issue_no)
+                break
+    return issues_number_list
+
+
+def inactiveIssue(request):
+    try:
+        GH_TOKEN = get_api_key('GH')
+    except:
+        return HttpResponse('[]')
+    g1 = Github(GH_TOKEN)
+    org = g1.get_organization('coala')  # Ignore KeywordBear
+    repo = org.get_repo('community')
```

Comment:
After getting the `org_name` get the list of all of its repo and run your code for each repo to get_inactive issues; store it along with its repo name and then print it.

Reply:
@sks444  Currently there is a reason for not doing this.
https://github.com/coala/community/pull/91#discussion_r174018094